src-tauri/capabilities/default.json

```diff
@@ -63,16 +63,25 @@
             "identifier": "http:allow-fetch",
             "allow": [
                 {
-                    "url": "https://*genius.com"
+                    "url": "https://*.genius.com"
```

Comment:
use `*.genius.com` instead of `*genius.com` since `zyxgenius.com` would also match.